tabpy-server/tabpy_server/__init__.py

```diff
@@ -1,4 +1,23 @@
-
-from ._version import get_versions
-__version__ = get_versions()['version']
-del get_versions
+import logging
+import logging.handlers
+import os
+import tempfile
+
+from ._version import get_versions
```

Comment:
Looks like you took some changes from our `dev` branch which are not merged to `master` yet. What we have in `master` is still uses versioneer code. I tried your changes (specifically the branch you are trying to merge from) and it doesn't work for me specifically for the versioning code.

Reply:
Hmm OK. So is "dev" in reality the main branch?

Reply:
Well, it's complicated... :)

We meant to use dev when working on new features so we can have iterative changes without risking breaking master. And for other contributors, they should just create merge requests to merge in master. For when dev is out of sync with master we will take care of that.

The idea is you shouldn't deal with all the merging and conflict resolving for something which is not approved to be in master yet. Unless you want to :)

Reply:
Hmm OK. Actually pulled this from master not dev so think that's a red herring, but I think it's a mistake on my end as to how I managed the merge conflicts with some of the larger changes being done recently. Should be able to look in more detail over next or two and resolve accordingly. Will ping back